packages/gotrue/lib/src/gotrue_client.dart

```diff
@@ -92,11 +103,13 @@ class GoTrueClient {
     Client? httpClient,
     GotrueAsyncStorage? asyncStorage,
     AuthFlowType flowType = AuthFlowType.pkce,
+    bool broadcastSession = true,
```

Comment:
I'm thinking we might not want to expose a parameter for the user to choose to turn this on or off, but is there a situation where the user might want to turn this off? Is this so that the users can keep the same behavior after this version release as well?

Reply:
I was unsure whether to add this or not. auth-js has the session persisting built in and not outsourced to supabase-js like we do with supabase_flutter. auth-js only broadcasts the events if session persistence is enabled. https://github.com/supabase/auth-js/blob/8222ee198a0ab10570e8b4c31ffb2aeafef86392/src/GoTrueClient.ts#L244 So I was thinking about doing it more like in js and managing the session persistence in gotrue-dart so that we only broadcast if session gets persisted. But that's to be discussed in another issue so I will remove the option here for now.